### PR TITLE
backport: bitcoin#19585, #19590, #19605, #19613, #19620, #19655, #19660, #27629, bitcoin-core/gui#14, #34

### PR DESCRIPTION
**Pull request by knst**
## Issue being fixed or feature implemented
Just regular bitcoin backports, mostly from v21

## What was done?
 - bitcoin-core/gui#34
 - bitcoin-core/gui#14
 - bitcoin/bitcoin#19613
 - bitcoin/bitcoin#19590
 - bitcoin/bitcoin#19585
 - bitcoin/bitcoin#27629
 - bitcoin/bitcoin#19620
 - bitcoin/bitcoin#19660
 - bitcoin/bitcoin#19605
 - bitcoin/bitcoin#19655

## How Has This Been Tested?
Run unit/functional tests

## Breaking Changes
N/A

## Checklist:
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: src/qt/guiconstants.h [hunk 1]
```diff
@@ -26,8 +26,8 @@ static const bool DEFAULT_SPLASHSCREEN = true;
 #define RGB_HALF 0x7f7f7f
 /** Path to the icon resource folder */
 #define ICONS_PATH ":icons/"
-/** Path to the movies resource folder */
-#define MOVIES_PATH ":movies/"
+/** Path to the animation resource folder */
+#define MOVIES_PATH ":animation/"
```
Comment (UdjinM6) [inline src/qt/guiconstants.h:30]: gui-14: maybe `s/MOVIES_PATH/ANIMATION_PATH/` for consistency (also in `src/qt/bitcoingui.cpp`)

Comment (UdjinM6) [reply src/qt/guiconstants.h]: dropping it was not smth I suggested 😅 but it's used in one single place only so I guess it's ok.

**Review by UdjinM6 — APPROVED** [review]
utACK

File: src/qt/guiconstants.h (hunk 1 above)
Comment (knst) [reply src/qt/guiconstants.h]: I thought to drop `ICONS_PATH` also but they used twice... so... I keep them

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Pull request merged**